test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -60,4 +65,13 @@ public static Person generateTestPerson() {
             return null;
         }
     }
+
+    /**
+     * Asserts whether the text in the two given files are the same.
+     */
+    public static void assertTextFilesEqual(Path path1, Path path2) throws IOException {
+        List<String> list1 = Files.readAllLines(path1, Charset.defaultCharset());
+        List<String> list2 = Files.readAllLines(path2, Charset.defaultCharset());
+        assertEquals(list1, list2);
```

Comment:
> but I think that to have a full content is clearer to see

Usually it's not. Since this is a utility method, it is better to cater to the general case.

But really, you can try and see for yourself and report back the results :-)

> if you only show the differences, you may not know which Person the differences belong to, i.e. if only Email differs, the other information is not shown right?

JUnit shows some context with the differences.